api/src/Entity/ContentNode.php

```diff
@@ -51,6 +51,9 @@
 #[ORM\InheritanceType('SINGLE_TABLE')]
 #[ORM\DiscriminatorColumn(name: 'strategy', type: 'string')]
 #[ORM\UniqueConstraint(name: 'contentnode_parentid_slot_position_unique', columns: ['parentid', 'slot', 'position'])]
+#[ORM\Index(columns: ['createTime'])]
+#[ORM\Index(columns: ['updateTime'])]
+#[ORM\Index(columns: ['strategy'])]
```

Comment:
I remember vaguely having looked into this earlier, at the conclusion back then was that indexes on discriminator columns are considered bad practice:
https://github.com/doctrine/orm/issues/2037

The issue is quite old, so not sure this is still true.